command/commands/user.py

```diff
@@ -155,6 +156,10 @@ def edit_helper(self, user_id, param_list):
             edited_user.set_position(param_list["pos"])
         if param_list["github"]:
             edited_user.set_github_username(param_list["github"])
+            try:
+                self.github.org_add_member(param_list["github"])
+            except Exception as e:
```

Comment:
Catching `Exception` is probably unnecessarily broad - I would catch only the `GithubAPIException` exposed by the `interface` module.